docker/bashrc.sh

```diff
@@ -5,6 +5,9 @@ alias dj="python manage.py"
 if [ "$BUILD_ENV" = "dev" ]; then
     alias djrun="python manage.py runserver 0.0.0.0:8000"
     alias djtest="python manage.py test --settings=tbx.settings.test"
+    alias djcov="coverage erase && \
+                coverage run --branch manage.py test --settings=tbx.settings.test && \
```

Comment:
We could also consider adding the coverage to CI - [see this example on rca](https://github.com/torchbox/rca-wagtail-2019/pull/962/files) - but don't let that block this mr.

Reply:
You can also see https://github.com/wagtail/wagtail-ai/pull/49 on how to add the coverage report to the GitHub Action summary